### ISSUE TITLE
Opt out mechanism from autosync

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/marketing#287]
We need to figure out how existing YPP members can opt out from
- Auto sync
- Programme

**Comment by dmtrjsg** [comment]
@bedeho is it correct to assume that opting out from auto-sync does not terminate the YPP programme participation?

I would say that these should be separated, but wanted to drive this via yourself before we start working on this on the design and tech side.

**Comment by bedeho** [comment]
Yes I think so, but admittdely my memory is a bit shaky here, we should probably have a YPP meeting in a meeting or two, just to refresh that all our goals make sense and if we should simply something, if only for operational purposes. It is already more complex than we originally thought, because we it would be launched without sync.

**Comment by dmtrjsg** [comment]
@zeeshanakram3 already created a draft PR (prototype) already with some considerable progress there.. based on the super rough and high level estimate remainder of the Sync may take about 2 weeks to bring to production level prototype to be tested. ⚠️ NB: this is a very rough indication just to draw the line in the sand.

PR link: https://github.com/Joystream/youtube-synch/pull/41

**Comment by dmtrjsg** [comment]
Afters peaking with Martin also adding a case not to forget:
- if channel gets deleted, opt out from both autosync and ypp should be implied
- channel owner can remove the keys from the channel (via polkadot.js app) which should mean an automatic opt out

**Comment by dmtrjsg** [comment]
+ 
I can see that someone would be interested in Remaining in YPP programme but with Sync disabled

- Remain on YPP without the sync to benefit from outbout marketing rewarded events posting Gleev videos to their channel and doing referrals but not YT-> Gleev rewards to retain exclusive content on YT or only post selected content to Gleev (perhaps fitting in to more narrow category structure in focus of Gleev)

I cannot see any benefits from being out of YPP but with Sync enabled, as this is just loosing free money on the table.
